src/subcommands/psa_generate_random.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Generates a sequence of random bytes.
+
+pub use crate::cli::ParsecToolApp;
+use crate::error::ParsecToolError;
+use crate::subcommands::common::ProviderOpts;
+use crate::subcommands::ParsecToolSubcommand;
+use parsec_client::core::interface::requests::ProviderID;
+use parsec_client::core::operation_client::OperationClient;
+use parsec_interface::operations::psa_generate_random;
+use parsec_interface::operations::{NativeOperation, NativeResult};
+use std::convert::TryFrom;
+use structopt::StructOpt;
+
+/// Generates a sequence of random bytes.
+#[derive(Copy, Clone, Debug, StructOpt)]
+#[structopt(name = "psa_generate_random")]
+pub struct PsaGenerateRandomSubcommand {
```

Comment:
Is it possible to have an optional argument for an output file and to store the random data in that file, in binary format?

Reply:
And/or an option to only print the random data (hex encoded) to stdout, maybe through that verbosity argument?

Reply:
Ooh that's a good idea, will do that. 😄